src/main/telemetry/crsf.c

```diff
@@ -297,6 +302,60 @@ void crsfFrameBatterySensor(sbuf_t *dst)
     sbufWriteU8(dst, batteryRemainingPercentage);
 }
 
+#if defined(USE_BARO) && defined(USE_VARIO)
```

Comment:
This should be OR, perhaps, or maybe better to just enable this sensor if USE_BARO is set.

Reply:
resolved?

Reply:
Yes it uses both functionalities